enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java

```diff
@@ -17,7 +17,12 @@
 
 package de.schildbach.pte;
 
-import static com.google.common.base.Preconditions.checkNotNull;
+import com.google.common.base.Strings;
```

Comment:
Re-organizing imports causes noisy commits. Is this necessary?

Reply:
Android Studio does this automatically for me :(
I'll remove them manually.